src/docs/self-hosted/backup.mdx

```diff
@@ -9,7 +9,8 @@ If you need a quick way to backup and restore your Sentry instance and you don't
 ### Backup
 
 ```shell
-docker-compose run --rm -T -e SENTRY_LOG_LEVEL=CRITICAL web export > sentry/backup.json
+// $(pwd) is interchangeable with the current filepath of self-hosted
```

Comment:
I think `pwd` is self-explanatory and we can move the comment outside the code block, like so:

```
Run this from your `self-hosted` directory:

\```
docker-compose run -v $(pwd)/sentry:/sentry-data/backup --rm -T -e SENTRY_LOG_LEVEL=CRITICAL web export /sentry-data/backup/backup.json
\```
```

Reply:
Fair enough. I'll move the comment above actually since both backup and restore will be run from the `self-hosted` directory